DataManagementSystem/Agent/FTSAgent.py

```diff
@@ -925,7 +925,10 @@ def __finalizeFTSJob( self, request, ftsJob ):
     if not processFiles["OK"]:
       log.error( processFiles["Message"] )
       return processFiles
-    ftsFilesDict = self.updateFTSFileDict( ftsFilesDict, processFiles["Value"] )
+    processFiles = processFiles['Value']
+    if processFiles['toRegister']:
+      log.error( "%f files could not be registered in FC" % len( processFiles['toRegister'] ) )
```

Comment:
This is indeed a non-existing requirement I guess… or a joke  ;-)))
I have been fixing hundreds of log calls with “dfsfd gssf g %s” % res[‘Message’] which I think makes sense, but reporting the number of items makes no sense in the way you write it, I am sorry… and it is totally useless unless the two items are reported separately in some magic DB somewhere???
If this is mandatory, don’t make the second argument optional… and fix all the calls throughout DIRAC…
Philippe

Le 19 janv. 2015 à 23:33, Andrei Tsaregorodtsev <notifications@github.com> a écrit :

> In DataManagementSystem/Agent/FTSAgent.py:
> 
> > @@ -925,7 +925,10 @@ def __finalizeFTSJob( self, request, ftsJob ):
> >      if not processFiles["OK"]:
> >        log.error( processFiles["Message"] )
> >        return processFiles
> > -    ftsFilesDict = self.updateFTSFileDict( ftsFilesDict, processFiles["Value"] )
> > +    processFiles = processFiles['Value']
> > +    if processFiles['toRegister']:
> > +      log.error( "%f files could not be registered in FC" % len( processFiles['toRegister'] ) )
> I remind your again that log error messages must have a static part and a variable part as the first and the second argument. So, this line should rather be:
> log.error( "A number of files could not be registered in FC", len( processFiles['toRegister'] ) )
> 
> —
> Reply to this email directly or view it on GitHub.
> 

-- 
Dr Philippe Charpentier
J09210, Physics Department, CERN, CH-1211 Genève 23
LHCb Experiment Distributed Computing Coordinator
Office 2-R-007, Tel : +41 22 767 4244 ,  GSM : +41 76 487 0167
mailto:Philippe.Charpentier@cern.ch  http://cern.ch/Philippe.Charpentier